sass/_c1-screenshots.scss

```diff
@@ -0,0 +1,94 @@
+// help from https://www.youtube.com/watch?v=sDsyHJnkZn0
+.underlined-heading-screenshot{
+  margin: 0 auto 4em auto;
+  padding: 0;
+  &-text {
```

Comment:
I appreciate the use of `&` to extend the classname here, but I think you could have solved this with a smaller amount of CSS (and the use of pseudo elements) as I outlined [earlier in the review](https://github.com/SEA-Design-Dev/project_week_one/pull/11/files#diff-528c3923d718a8860f5d8c05c3931c55R62).